wiki/PlayersManual.md

```diff
@@ -350,25 +350,35 @@ Trading panel: | &nbsp;
 :--- | :---
 Buy | +, Return, or Space
 Sell | -, Delete, or Backspace
-Buy / sell 5x | Shift
-Buy / sell 20x | Control / Command
-Buy / sell 500x | Option / Alt
 Buy / sell all | Shift + B / S
 View commodity price map | Map key
 
-The multiplier modifiers (Shift = 5x, Command / Control = 20x, and Option / Alt = 500x) also work in the hiring panel, the shipyard, and the outfitter. And, they can be combined together - for example, Shift + Control to buy or sell 100 items at a time.
+Multipliers: | &nbsp;
+:--- | :---
+5x | Shift
+20x | Control / Command
+500x | Option / Alt
+
+The multiplier modifiers work in the Trading panel, Hiring panel, Shipyard and Outfitter. They can be used in combination - for example, Shift + Control to buy or sell 100 items at a time. All three used together will provide a whopping 50,000x multiplier.
 
 In addition to the multiplier keys, there are various other shortcuts in the outfitter:
 
-Outfitter: | &nbsp;
-:--- | :---
-Buy | B
-Sell | S
-Send to cargo | C
-Install | I
-Uninstall | U
+Outfitter: | &nbsp; | First tries: | Then: | And finally:
+:--- | :--- | :--- | :--- | :---
+Buy | B | Install\* from cargo to ship | Install\* from storage to ship | Buy from outfitter to ship
+Sell | S | Sell from cargo | Sell from storage | Sell from ship
+Uninstall | U | Uninstall from ship to cargo | Uninstall, overflow to storage | &nbsp;
+Install | I | Install to ship from cargo | Install to ship from storage | &nbsp;
+Cargo | C | Transfer\* to cargo from planetary storage | Buy from outfitter to planetary storage | &nbsp;
```

Comment:
```suggestion
Cargo | C | Transfer to cargo from planetary storage | Buy from outfitter to planetary storage | &nbsp;
```

Reply:
See latest updates, Buy is just: Buy from outfitter and install on ship
This matches the flow diagram I shared with [#10743](https://github.com/endless-sky/endless-sky/pull/10743)